praw/models/listing/mixins/subreddit.py

```diff
@@ -8,14 +9,19 @@
 from .gilded import GildedListingMixin
 from .rising import RisingListingMixin
 
+Comment = TypeVar("Comment")
+_CommentHelper = TypeVar("_CommentHelper")
```

Comment:
Ordering or _ removal.

Reply:
Understood

Reply:
Now that I understand why the underscores sometimes are necessary, it seems in this case it would be better to define `CommentHelper` before `SubredditListingMixin`. That also happens to be the sort order the classes _should_ be in.

Reply:
I presumed that the sort order was intentional. Should I define commenthelper before the mixin?

Reply:
Yes, I think it's preferable to reorder things in order to prevent needing to to define a new global variable.